src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -106,11 +105,20 @@ class _SessionStatusValue:
 _VirtualColumnName = Literal["value", "avg", "max", "p50", "p75", "p90", "p95", "p99"]
 
 
-def _to_column(query_func: SessionsQueryFunction) -> SelectableExpression:
+def _to_column(
+    query_func: SessionsQueryFunction, column_condition: Optional[Function] = None
+) -> SelectableExpression:
     """
     Converts query a function into an expression that can be directly plugged into anywhere
     columns are used (like the select argument of a Query)
     """
+
+    suffix = ""
+    parameters = [Column("value")]
+    if column_condition is not None:
```

Comment:
you can construct `uniqIf(..., 1)` instead of `uniq(...)` and save some complexity by getting rid of `suffix` that way

Reply:
Done!